.github/workflows/frontend-regression-tests.yml

```
@@ -0,0 +1,33 @@
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        node-version: [20.x, 22.x]
+    
+    defaults:
+        run:
+            working-directory: frontend
+    
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)